src/main/java/dev/dfonline/codeclient/Event.java

```diff
@@ -52,10 +54,14 @@ public static <T extends PacketListener> void handlePacket(Packet<T> packet) {
 
     public static <T extends PacketListener> void onSendPacket(Packet<T> packet) {
         if (packet instanceof CommandExecutionC2SPacket command) {
+            CommandSender.registerCommandSend();
```

Comment:
Just in case, are you able to confirm that command execution doesnt also trigger chat message packet (so we don't increment it twice)

Reply:
it does not increment twice